curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/build-javascript-objects.english.md

```diff
@@ -67,21 +67,11 @@ tests:
 <div id='js-seed'>
 
 ```js
-// Example
-var ourDog = {
-  "name": "Camper",
-  "legs": 4,
-  "tails": 1,
-  "friends": ["everything!"]
-};
```

Comment:
This `ourDog` example doesn't seem to be placed back in the challenge description instead, unlike other similar challenges in the PR. Is it because this particular challenge already has 2 similar examples showing the usage (`cat` and `anotherObject`)?

Reply:
Yes, I decided to remove this one, since the Description already had an example.